pkg/spec/spec.go

```diff
@@ -30,14 +31,14 @@ type Function struct {
 
 // FunctionSpec contains func specification
 type FunctionSpec struct {
-	Handler  string   `json:"handler"`
-	Function string   `json:"function"`
-	Runtime  string   `json:"runtime"`
-	Type     string   `json:"type"`
-	Topic    string   `json:"topic"`
-	Deps     string   `json:"deps"`
-	Memory   string   `json:"memory"`
-	Env      []string `json:"env"`
+	Handler  string            `json:"handler"`
+	Function string            `json:"function"`
+	Runtime  string            `json:"runtime"`
+	Type     string            `json:"type"`
+	Topic    string            `json:"topic"`
+	Deps     string            `json:"deps"`
+	Memory   resource.Quantity `json:"memory"`
```

Comment:
Instead, embed a PodSpec here somewhere, and use that to contain Memory and Env (and whatever we think of next since it's bound to grow to include just about all of PodSpec).